ReactiveCocoa/UIKit/iOS/UIPickerView.swift

```diff
@@ -0,0 +1,43 @@
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+private class PickerViewDelegateProxy: DelegateProxy<UIPickerViewDelegate>, UIPickerViewDelegate {
+	@objc func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+		forwardee?.pickerView?(pickerView, didSelectRow: row, inComponent: component)
+	}
+}
+
+extension Reactive where Base: UIPickerView {
+
+	private var proxy: PickerViewDelegateProxy {
+		return .proxy(for: base,
+		              setter: #selector(setter: base.delegate),
+		              getter: #selector(getter: base.delegate))
+	}
+
+	/// Sets the selected row in the specified component, without animating the
+	/// position.
+	public func selectedRow(inComponent component: Int) -> BindingTarget<Int> {
+		return makeBindingTarget { $0.selectRow($1, inComponent: component, animated: false) }
+	}
+
+	/// Reloads all components
+	public var reloadAllComponents: BindingTarget<()> {
+		return makeBindingTarget { base, _ in base.reloadAllComponents() }
+	}
+
+	/// Reloads the specified component
+	public var reloadComponent: BindingTarget<Int> {
+		return makeBindingTarget { $0.reloadComponent($1) }
+	}
+
+	/// Create a signal which sends a `value` event for each row selection
+	///
+	/// - returns:
+	///   A trigger signal.
+	public var selections: Signal<(Int, Int), NoError> {
```

Comment:
Would it be better to add a `Selection` type for this?

```swift
struct UIPickerViewSelection {
    var row: Int
    var component: Int
}

public var selections: Signal<UIPickerViewSelection, NoError> { ... }
```

(That's an honest question.)

I find it a little strange that UIKit doesn't use an `NSIndexPath` for this. Maybe we should just adopt that.